rfhub2/cli/client.py

```diff
@@ -0,0 +1,79 @@
+from requests import session, post, get, delete, exceptions
+from typing import Dict
+from rfhub2.model import Collection, Keyword
+
+PROTOCOL = 'http://'
+API_V1 = 'api/v1'
+TEST_COLLECTION = {"name": "a", "type": "a", "version": "a", "scope": "a",
+                   "named_args": "a", "path": "a", "doc": "a", "doc_format": "a"}
+
+
+class Client(object):
+    """
+    API client with methods to populate rfhub2 application.
+    """
+    def __init__(self, app_interface: str, port: int, user: str, password: str,):
+        self.app_interface = app_interface
+        self.port = port
+        self.auth = (user, password)
+        self.session = session()
+        self.app_url = f'{PROTOCOL}{self.app_interface}:{self.port}'
+        self.api_url = f'{self.app_url}/{API_V1}'
+
+    def check_communication_with_app(self) -> None:
+        try:
+            self.get_collections()
+        except exceptions.RequestException:
+            print(f'Connection to application at {self.app_url} refused!\n'
+                  f'Check parameters for app-interace and port.')
+            exit(1)
```

Comment:
Again, I would prefer raising error with meaningful message without manual exiting. It will be easier to test.